lib/__tests__/standalone.test.mjs

```diff
@@ -466,3 +466,95 @@ describe('existing codeFilename for nested config detection', () => {
 		expect(results[0].warnings).toHaveLength(1);
 	});
 });
+
+describe('standalone with fix options for CSS syntax errors', () => {
```

Comment:
[suggestion] It seems reasonable that this case would be added to `standalone-fix.test.mjs`. Can you move it?

Reply:
Fixed in [this commit](https://github.com/stylelint/stylelint/pull/8106/commits/caccdc943017ecb2b198b12908eb33c3ed4e2317).